### PR TITLE
Update tutorial first page dashboard image

### PR DESCRIPTION
**Pull request by axa99**
## Description

In the 'Get started with your first dashboard' tutorial page, the result for the 'Add first component' shows an image of the dashboard with a filter control.  But the app.py code for 'First Component' does not include the relevant code for the filter control.  

To make the 'Result' image consistent with the code, the dashboard21.png image was updated.  The updated image does not show the filter component on the dashboard and is consistent with the app.py code for the 'First Component'

## Checklist

- [x] I have not referenced individuals, products or companies in any commits, directly or indirectly
- [x] I have not added data or restricted code in any commits, directly or indirectly
- [x] I have updated the docstring of any public function/class/model changed
- [ ] I have added the PR number to the change description in the changelog fragment, e.g. `Enable feature XXX ([#1](https://github.com/mckinsey/vizro/pull/1))` (if applicable)
- [ ] I have added tests to cover my changes (if applicable)

## Types of changes

- [x] Docs/refactoring (non-breaking change which improves codebase)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

  - I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
  - I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorised to submit this contribution on behalf of the original creator(s) or their licensees.
  - I certify that the use of this contribution as authorised by the Apache 2.0 license does not violate the intellectual property rights of anyone else.

### CODE REVIEW

File: vizro-core/docs/assets/tutorials/dashboard/dashboard21.png
Comment (maxschulz-COL) [inline vizro-core/docs/assets/tutorials/dashboard/dashboard21.png:1]: Great spot! The shown image is indeed with a filter that does not exist in the configuration.

In fact, in the meantime ([#68](https://github.com/mckinsey/vizro/pull/68)) introduced an additional feature that would remove the entire side panel if there is only one page and no controls, so the next release would actually include the following behaviour:

image

If you're happy to change the new picture to the below, then we can move ahead 💪

**Review by maxschulz-COL — COMMENTED** [review]
Thank you very much for your contribution 🙏  This is very much appreciated and a very valuable contribution to Vizro!

I have provided feedback and activated the tests, let me know when I should re-review.

Many thanks again for your valuable contirbution!

File: vizro-core/docs/assets/tutorials/dashboard/dashboard21.png
Comment (axa99) [reply vizro-core/docs/assets/tutorials/dashboard/dashboard21.png:1]: For sure! I made an update so that it uses the image with the side panel removed

**Review by huong-li-nguyen — COMMENTED** [review]
Hey @axa99 , 

you would have to create a changelog file for the linting CI to pass. Just run:

`hatch run docs:changelog `

and commit the file as is. And don't forget to add yourself as a contributor here under **Previous team members and code contributors**: https://github.com/mckinsey/vizro/blob/main/vizro-core/docs/pages/development/authors.md

Let me know if you have any issues with the hatch environment, see the contribution guide [here](https://github.com/mckinsey/vizro/blob/main/vizro-core/docs/pages/development/contributing.md).

**Comment by axa99** [comment]
@huong-li-nguyen I just pushed a commit with the changelog file added. Thanks!

**Comment by huong-li-nguyen** [comment]
> @huong-li-nguyen I just pushed a commit with the changelog file added. Thanks!

Hey @axa99, thanks a lot for your contribution! We hope you enjoyed the tutorial. We are always happy to receive feedback, so if you have any new ideas and suggestions, please feel free to share :)

**Review by Anna-Xiong — APPROVED** [review]

**Pull request merged**